react/components/Table/SimpleTable.js

```diff
@@ -3,11 +3,16 @@ import PropTypes from 'prop-types'
 import { MultiGrid, AutoSizer } from 'react-virtualized'
 import ArrowDown from '../icon/ArrowDown'
 import ArrowUp from '../icon/ArrowUp'
+import OptionsDots from '../icon/OptionsDots'
 import EmptyState from '../EmptyState'
 import Spinner from '../Spinner'
+import ActionMenu from '../ActionMenu'
+
 const ARROW_SIZE = 11
 const HEADER_HEIGHT = 36
 const DEFAULT_COLUMN_WIDTH = 200
+const LINE_ACTIONS_COLUMN_WIDTH = 70
+const NO_TITLE_COLUMN = ' '
```

Comment:
not sure why this needs to be an external variable, but ok

Reply:
or why this is a whitespace instead of `''`

Reply:
@lbebber when rendering the column, table checks for the title with something like `if (title)` so  `''` would be considered null

Reply:
@lbebber I don't like this behaviour too, I'll check with @ohmyguigs if it makes sense to change this 😃